GenderPayGap.WebUI/Areas/Account/Views/ChangeDetails/ChangeDetails.cshtml

```diff
@@ -13,7 +13,7 @@
 
             @Html.AntiForgeryToken()
 
-            <fieldset>
+            <fieldset class="visually-hidden">

```

Comment:
This should be `style="display: none;"` rather than `class="visually-hidden"`.

`visually-hidden` hides the text on the screen, but it's still there for screen readers to see
`display: none;` hides it for everyone